tests/client_request.rs

```diff
@@ -440,9 +440,14 @@ fn request_with_connection_headers() {
     let _ = ::env_logger::try_init();
     let (io, srv) = mock::new();
 
-    let srv = srv.assert_client_handshake()
+    // can't assert full handshake, since client never send a request, and
+    // thus never boths to ack the settings...
```

Comment:
client never _sends_ a request, and thus never _bothers_ ?

Reply:
Yea, currently the client includes the ack with the first request.

Reply:
mostly just noting some typos ;)